cts/cli/regression.rules.exp

```diff
@@ -91,6 +92,7 @@ No rule found with ID=blahblah
 * Passed: crm_rule       - Try to check a rule that doesn't exist
 =#=#=#= Begin test: Try to check a rule that doesn't exist, with XML output =#=#=#=
 <pacemaker-result api-version="X" request="crm_rule -c -r blahblah --output-as=xml">
+  <rule-check rule-id="blahblah" rc="105"/>
```

Comment:
@clumens Example of new XML output when we call out->message in the event of rule-check errors too.

One advantage is that I think previously, the status code would be the last error we found. Now we get a discrete rule-check element and return code for every rule we checked.

Reply:
That does seem helpful. Need to update the xml/api/crm_rule schema and bump PCMK__API_VERSION

Reply:
zeroOrMore to oneOrMore rule-checks? Can't think of what else might need to change

Reply:
New schemas should remain compatible with output produced by earlier versions, so zeroOrMore

Reply:
I'm saying I don't see what in the schema needs to be updated for this PR. (There is every chance in the world I've got a glaring blind spot right now.)

Reply:
My bad, I thought this was completely new but it's just a new occurrence

Reply:
Ah gotcha, no worries